api/config.py

```diff
@@ -52,27 +52,28 @@
     },
 }
 
-__config = copy.deepcopy(DEFAULT_CONFIG)
+def apply_env_variables(config):
+    # Overwrite default config values with SCITRAN env variables if available
+    for outer_key, scoped_config in config.iteritems():
+        if outer_key in ['core', 'site', 'auth', 'persistent']:
```

Comment:
I also call it [here](https://github.com/scitran/core/pull/295/files/a935657d5a852fc569c6394c2ae0c7de3e7e71a7#diff-283256b6a6d2885d51431bd002292cdaR199) to preserve any environment variables on startup after applying configs from the db. I could also check if `scoped_config` has an `__iter__` property so we don't try to iterate over the dates.